CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 - Add getters for `Validator` object to expose more data to users.
 - Add test file for `Validator` object.
 
+- Add `networkId` to `WalletData` so that it is saved with the seed data and surfaced via the export function
```

Comment:
This should get added to the unreleased section, since it'll be a part of 0.13.0 when we release

Reply:
oops, must have shifted around in the rebase. will update!